projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -194,13 +194,6 @@ function grunion_message_bulk_spam() {
 	echo '<div class="updated"><p>' . __( 'Feedback(s) marked as spam', 'jetpack' ) . '</p></div>';
 }
 
-// remove admin UI parts that we don't support in feedback management
```

Comment:
Removed this piece of code since it seemed obsolete. After loading this file, we hook again into `admin_menu` for registering the feedback menu and submenus.